unified-codes/apps/web/src/components/layout/ErrorLayout.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
```

Comment:
A bit unnecessary here, but for the sake of consistency and to keep a common pattern going! Will probably find a better layout when more error pages are added.

Reply:
Oh soz. I whined about this elsewhere.
Yes, agree re the consistency - though I would have ditched consistentness in this case, it works and is elegant. Ignore previous comment 😉